Pipelines/Templates/tests.yaml

```diff
@@ -14,61 +9,33 @@ parameters:
     - StandaloneWindows64
     - WSAPlayer
     - Android
-
+  default: StandaloneWindows64
+    
 - name: PathToProject
   type: string
+  default: ''
```

Comment:
I think I'm fine adding the default value for the platform, but I don't think this template should accept an empty string as a default value for the project path.

Reply:
Ah yes, I can remove that default value now, was used for testing purposes at one point.